R/cube-result.R

```diff
@@ -13,6 +13,130 @@ setMethod("initialize", "CrunchCube", function (.Object, ...) {
     return(.Object)
 })
 
+#' @rdname cube-methods
+#' @export
+
+setMethod("[", "CrunchCube", function (x, i, j, ..., drop = TRUE) {
+    subset <- eval(substitute(alist(i, j, ...)))
+    subset <- replaceMissingWithTRUE(subset)
+    translated_subset <- translateCubeIndex(x, subset, drop)
+
+    out <- x
+    out@arrays$count <- subsetByList(out@arrays$count, translated_subset, drop)
```

Comment:
`lapply` over `out@arrays`, there may be many measures (e.g. via `crplyr::summarize()`)